packages/scene-composer/src/components/panels/SceneNodeInspectorPanel.tsx

```diff
@@ -168,6 +212,7 @@ export const SceneNodeInspectorPanel: React.FC = () => {
               values={selectedSceneNode.transform.rotation}
               toStr={(a) => THREE.MathUtils.radToDeg(a).toFixed(3)}
               fromStr={(s) => THREE.MathUtils.degToRad(toNumber(s))}
+              disabled={isTagComponent || isOverlayComponent ? [true, true, true] : undefined}
```

Comment:
Can this return `false` instead of `undefined`?

Reply:
we will need `[false, false, false]`

Reply:
We should make that update then. I know there's a lot of use of `undefined`, but we should avoid introducing it unnecessarily. Type consistency FTW!

Reply:
I think it's a common thing to use the default value by passing `undefined` as a value for optional parameters

Reply:
This one is interesting. I'm not going to weigh in on the `undefined`/`not undefined` topic, but I'm more curious about the datastructure `[true, true, true]`. This is not what I'd expect to have to pass to a `disabled` property, and feels like a violation of single responsibility principle. Maybe these 3 true statements (if they are individually important) be collapsed into a standard boolean (true?) or should they be 3 separate props and let the component sort it out?

Reply:
it is to disable X, Y, Z individually. we use triple for values like position as well